src/igvfd/types/file_set.py

```diff
@@ -86,6 +86,34 @@ class MeasurementSet(FileSet):
     item_type = 'measurement_set'
     schema = load_schema('igvfd:schemas/measurement_set.json')
 
+    @calculated_property(
+        condition='multiome_size',
+        schema={
+            'title': 'Related Multiome Datasets',
+            'description': 'Related datasets included in the multiome experiment this measurement set is a part of.',
+            'type': 'array',
+            'uniqueItems': True,
+            'items': {
+                'title': 'Related Multiome Dataset',
+                'description': 'Related dataset included in the multiome experiment this measurement set is a part of.',
+                'type': 'string',
+                'linkTo': 'MeasurementSet'
+            },
+            'notSubmittable': True,
+        }
+    )
+    def related_multiome_datasets(self, request, accession, samples=None):
+        related_datasets = []
+        if samples:
+            for sample in samples:
+                sample_object = request.embed(sample, '@@object')
+                for file_set in sample_object['file_sets']:
```

Comment:
if sample_object has no 'file_sets' you will get an exception here. probably should use "get"

Reply:
I will add this, but sample_object should always have 'file_sets' since it's reverse linked and this is checking all samples on a given measurement_set.

Reply:
it is status filtered and better to assume as little as possible